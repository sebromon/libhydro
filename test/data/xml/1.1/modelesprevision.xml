<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2001-12-17T04:30:47</DateHeureCreationFichier>
    <RefFichier>modele.xml</RefFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">825</CdIntervenant>
      <CdContact schemeAgencyID="SCHAPI">222</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
      <CdContact schemeAgencyID="SCHAPI">2</CdContact>
    </Destinataire>
  </Scenario>
  <RefHyd>
    <ModelesPrevision>
      <ModelePrevision>
        <CdContact>1234</CdContact>
        <CdModelePrevision>9876543210</CdModelePrevision>
        <LbModelePrevision>Libellé du modèle</LbModelePrevision>
        <TypModelePrevision>1</TypModelePrevision>
        <DescModelePrevision>Description du modèle</DescModelePrevision>
        <DtMajModelePrevision>2001-12-17T04:30:47</DtMajModelePrevision>
      </ModelePrevision>
      <ModelePrevision>
        <CdModelePrevision>0123456789</CdModelePrevision>
      </ModelePrevision>
    </ModelesPrevision>
  </RefHyd>
</hydrometrie>
