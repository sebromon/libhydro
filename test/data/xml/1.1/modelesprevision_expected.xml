<?xml version="1.0" encoding="UTF-8"?>
<hydrometrie xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd">
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2001-12-17T04:30:47</DateHeureCreationFichier>
    <!-- <RefFichier>modele.xml</RefFichier> -->
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">825</CdIntervenant>
      <CdContact>222</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
      <CdContact>2</CdContact>
    </Destinataire>
  </Scenario>
  <RefHyd>
    <ModelesPrevision>
      <ModelePrevision>
        <CdContact>1234</CdContact>
        <CdModelePrevision>9876543210</CdModelePrevision>
        <LbModelePrevision>Libellé du modèle</LbModelePrevision>
        <TypModelePrevision>1</TypModelePrevision>
        <DescModelePrevision>Description du modèle</DescModelePrevision>
        <DtMajModelePrevision>2001-12-17T04:30:47</DtMajModelePrevision>
      </ModelePrevision>
      <ModelePrevision>
        <CdModelePrevision>0123456789</CdModelePrevision>
        <TypModelePrevision>0</TypModelePrevision>
      </ModelePrevision>
    </ModelesPrevision>
  </RefHyd>
</hydrometrie>
